Tasks/Koval/Task_3/Task_3_1.py

```diff
@@ -0,0 +1,53 @@
+# 1. Написать программу для поиска наиболее часто встречающихся элементов в строке.
+# Пользователь вводит строку, и количество элементов, которые он хотел бы увидеть в ответе.
+
+# Пример:
+# Строка:lkseropewdssafsdfafkpwe
+# Элементы:3
+# 's' встречается 4 раза
+# 'e' встречается 3 раза
+# 'f' встречается 3 раза
+
+s = input('Input your sentence here: ')
+while True:
+    count_output_elem = input ('How many element you want see? ')
+    
+    count_output_elem = count_output_elem.replace(' ', '')
+    if not count_output_elem.isdigit():
+        print ('Please input correct digit')
+        continue
+    count_output_elem = int(count_output_elem)
+
+    d = {}
+
+    # Make count of elem
+    for c in s:
+        if d.get(c, '') == '':
```

Comment:
good choice, consider also the following example:
d.get(c, 0)
d[c] += 1
logic is the same, but realization is more simple